util/shell_exec.go

```diff
@@ -114,7 +114,7 @@ func (x Executor) Start() error {
 
 // Log verbosely logs the command.
 func (x Executor) Log(tag string) {
-	color.Set(color.FgYellow)
+	color.Set(color.FgMagenta)
```

Comment:
Was this was already merged?

Reply:
I made that change here, then created a separate branch, and didn't pull it back out. This change made it into v2.